.github/workflows/cypress-tests.yml

```
@@ -0,0 +1,30 @@
+name: Cypress Tests
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10 
+    env:
+      WORKING_DIR: ./E2E_tests
+      CYPRESS_NO_COMMAND_LOG: 1  
+      CYPRESS_VIDEO: false       
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4  
+      - name: Setup Node.js (with cache)
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22.17.1
```

как вариант, можно попробовать лочить мажорную lts версию
```suggestion
          node-version: 22
```

---

А надо? Во-первых, путаница. Во-вторых, только что вписал .nvmrc-файл и поправил реадми.  Объяснишь зачем указывать иную версию?